frontend/src/forms/ComponentLike/Probes.tsx

```diff
@@ -119,7 +119,6 @@ class RenderProbe extends React.PureComponent<Props> {
               name={`${name}.httpGet.host`}
               component={RenderNestedTextfield}
               placeholder="0.0.0.0"
-              validate={validateHostWithWildcardPrefix}
```

Comment:
Why delete this validator? Even this field is optional, you should also consider to validate context when it's not empty.